src/cookbooks/bigquery/access.md

```diff
@@ -1,82 +1,94 @@
+
 ## Table of Contents
+
 <!-- toc -->
 
 ## Interfaces
 
-You can access BigQuery datasets and tables by selecting any of the following methods:
-
-- [re:dash](./querying.md#from-redash)
-- [GCP BigQuery Console](./querying.md#gcp-bigquery-console)
-    - For advanced use cases including managing query outputs, table management. Requires GCP access to be granted by Data Operations.
-- [GCP BigQuery API Access](./querying.md#gcp-bigquery-api-access)
-    - For advanced use cases including automated workloads, ETL, [BigQuery Storage API](https://cloud.google.com/bigquery/docs/reference/storage/). Requires GCP access to be granted by Data Operations.
-    - Allows access to BigQuery via [`bq` command-line tool](https://cloud.google.com/bigquery/docs/bq-command-line-tool)
-- [Spark](./querying.md#from-spark)
-    - [Databricks](./querying.md#on-databricks)
-    - [Dataproc](./querying.md#on-dataproc)
-- [Colaboratory](./querying.md#from-colaboratory)
-
-Note that with the exception of Redash, all methods require access to be provisioned (see the next section).
-
-## Access Request
+### STMO (`sql.telemetry.mozilla.org`)
 
-For access to BigQuery when using the GCP Console and API, [file a bug](https://bugzilla.mozilla.org/enter_bug.cgi?assigned_to=jthomas%40mozilla.com&bug_file_loc=https%3A%2F%2Fmana.mozilla.org%2Fwiki%2Fx%2FiIPeB&bug_ignored=0&bug_severity=normal&bug_status=NEW&bug_type=task&cf_fx_iteration=---&cf_fx_points=---&comment=Please%20grant%20me%20access%20to%20the%20BigQuery%20GCP%20console%20and%20API%20Access.%20I%20work%20on%20%3Cteam%3E.%0D%0A%0D%0AMy%20mozilla.com%20ldap%20login%20is%20%3Cyour%20ldap%20login%3E%40mozilla.com.&component=Operations&contenttypemethod=list&contenttypeselection=text%2Fplain&defined_groups=1&flag_type-4=X&flag_type-607=X&flag_type-800=X&flag_type-803=X&flag_type-936=X&form_name=enter_bug&maketemplate=Remember%20values%20as%20bookmarkable%20template&op_sys=Unspecified&priority=--&product=Data%20Platform%20and%20Tools&qa_contact=jthomas%40mozilla.com&rep_platform=Unspecified&short_desc=BigQuery%20GCP%20Console%20and%20API%20Access%20for%20%3Cyour%20ldap%20login%3E%40mozilla.com&target_milestone=---&version=unspecified). As part of this request we will add you to the appropriate Google Groups and provision a GCP Service Account.
+All users with access to [STMO](../../tools/stmo.md) can access BigQuery using the following data sources:
 
-## From re:dash
-
-All Mozilla users can access BigQuery using [re:dash](https://sql.telemetry.mozilla.org/) through the following Data Sources:
 - `Telemetry (BigQuery)`
 - `Telemetry Search (BigQuery)`
-    - This group is restricted to users in the re:dash `search` group.
 
-Access via re:dash is read-only. You cannot create views or tables using re:dash.
+Note that queries made from STMO are read-only: you cannot create views or tables.
```

Comment:
I'm going to leave this one as-is: the "therefore" doesn't quite read right to me as it implies causality rather than identity. Wrt the note part, we can actually use a blockquote which looks a little better than what you suggest, I'll implement this before landing.